src/modules/layout/components/Header.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import Headroom from 'react-headroom'
+import style from './Header.css'
+import logoWide from '../assets/integreat-app-logo.png'
+import HeaderMenuBar, { MENU_ITEMS_PROP_TYPE } from './HeaderMenuBar'
+import HeaderActionBar, { ACTION_ITEMS_PROP_TYPE } from './HeaderActionBar'
+
+/**
+ * The standard header which can supplied to a Layout. Displays a logo left, a HeaderMenuBar in the middle and a
+ * HeaderActionBar at the right (RTL: vice versa). On small viewports the HeaderMenuBar is shown underneath the rest
+ * of the Header.
+ * Uses Headroom to save space when scrolling.
```

Comment:
Did we verify headroom already? I'm my tests its responsible for more than 10ms delay for every page switch.

Reply:
What do you mean by verify?
I think we should create a task if we should continue using Headroom or write it ourselves in a better way, but I'd prefer having 10ms delay and that feature over not having 10ms delay and not having that feature :D

Reply:
Just wanted to remind that we still have the performance bug. My guess is that it's because of Headroom.